### ISSUE TITLE
No connection to remote Jupyterhub via API token with remote file system

### ISSUE THREAD

**Issue by RGBYCP** [microsoft/vscode-jupyter#9771]
### Applies To

- [X] Notebooks (.ipynb files)
- [ ] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

### What happened?

I have access to a remote jupyter server where interactive sessions are created in the browser. A standard conda kernel is offered or the user can select their own kernels based on their own conda environments. Jupyter notebooks and data are on this server.
I tried to generate a token and connect with VSC code via `https://remote-server.domain/user/username/?token=token_id`. 
I tried multiple times. Only once or twice, the remote jupyter server was recognised, but still no kernel could be selected. In the majority of tries nothing happened, not even a connection to the remote server. 

Idea is to get access to the remote Jupyterhub via VSC code and be able to use all nice extensions while coding in Jupyter in VSC code. 

### VS Code Version

Version: 1.66.2 (Universal)

### Jupyter Extension Version

v2022.3.1000901801

### Jupyter logs

```shell
I don't understand this unfortunately.
```

### Running Jupyter locally or remotely?

Remote

**Comment by RGBYCP** [comment]
After multiple tries today, I managed to connect to the remote Jupyter hub via VSC at least once. Maybe one should not touch the tokens beforehand, generate a new one and use the new one. The Jupyter hub kept crashing out of the blue, but I don't know if this is related.

My last question would be: I cannot see the notebooks on the remote Jupyter hub when connecting to the remote Jupyter hub. I can get only the remote kernel.
Is there a chance to change this, please? Or am I still missing something? 
Many thanks.

I will observe how stable the connection is and if future connections will work.

**Comment by DonJayamanne** [comment]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/1366
